scripts/test-setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# For testing CodaLab in Chrome
+wget https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.zip
+unzip chromedriver_linux64.zip
+sudo mv chromedriver /usr/bin/chromedriver
+sudo chown root:root /usr/bin/chromedriver
+sudo chmod +x /usr/bin/chromedriver
+
```

Comment:
Not super important, but it would be nice for setup if both zip files were deleted at the end.